test/commands/mode.js

```diff
@@ -81,10 +101,29 @@ describe("Commands", function () {
 
 			ModeCommand.input(testableNetwork, channel, "devoice", ["xPaw"]);
 			expect(testableNetwork.lastCommand).to.equal("MODE #thelounge -v xPaw");
+		});
+
+		it("should use ISUPPORT MODES on shorthand commands", function () {
+			ModeCommand.input(testableNetwork, channel, "voice", ["xPaw", "Max-P"]);
+			expect(testableNetwork.lastCommand).to.equal("MODE #thelounge +vv xPaw Max-P");
+
+			// since the limit for modes on tests is 4, it should send two commands
+			ModeCommand.input(testableNetwork, channel, "devoice", [
+				"xPaw",
+				"Max-P",
+				"hey",
+				"idk",
+				"thelounge",
+			]);
+			expect(testableNetwork.lastCommand).to.equal("MODE #thelounge -v thelounge");
+		});
+
+		it("should fallback to default limit of 1 mode for shorthand commands", function () {
+			ModeCommand.input(testableNetworkNoSupports, channel, "voice", ["xPaw"]);
+			expect(testableNetworkNoSupports.lastCommand).to.equal("MODE #thelounge +v xPaw");
 
-			// Multiple arguments are supported, sent as separate commands
-			ModeCommand.input(testableNetwork, channel, "devoice", ["xPaw", "Max-P"]);
```

Comment:
based on https://defs.ircdocs.horse/defs/isupport.html there should be no limit if `MODES` is undefined, not 1, but that could be inaccurate. Did you choose 1 for a reason?

Reply:
I chose 1 mostly for safety. Since I don't know whether the IRCd will accept all my modes or not (and since there's no mechanism for checking up if all were indeed set), I went that way. I can change that to no limit if desired, though.